tools/semgrep-py.yml

```diff
@@ -216,6 +216,9 @@ rules:
       - pattern-not-inside: |
           def get_object_from_key(...):
             ...
+      - pattern-not-inside: |
+          def realm_import_status(...):
+            ...
```

Comment:
Again you should be able to skip this by instead adding an option to `get_object_from_key`

Reply:
Done.